delivery_postlogistics/postlogistics/web_service.py

```diff
@@ -434,6 +445,13 @@ def get_access_token(cls, picking_carrier):
             cls.access_token_expiry = now + timedelta(seconds=response["expires_in"])
             return cls.access_token
 
+    def _sanitize_string(self, value):
+        """Removes disallowed chars ("|", "\", "<", ">") from strings."""
+        if isinstance(value, str):
```

Comment:
if we have False in there, do we need to return False or an empty string?

Reply:
The idea is to remove  disallowed chars from incoming strings, I don't think we should alter the value if it's something else than a string. That case should already be handled by the base implementation.

Reply:
I mean, sending `{"whatever": null}` and `{"whatever": ""}` is not the same, and I don't want to modify something that is currently working.